examples/large_models/deepspeed/Readme.md

```diff
@@ -44,3 +44,12 @@ torchserve --start --ncs --model-store model_store --models opt.tar.gz
 ```bash
 curl  "http://localhost:8080/predictions/opt" -T sample_text.txt
 ```
+
+### Running using TorchServe Docker Image
+
+To use DeepSpeed with GPU on TorchServe Docker, we need to build an image with [NVIDIA CUDA dev ](https://github.com/NVIDIA/nvidia-docker/wiki/CUDA) as the base image as shown [here](https://github.com/pytorch/serve/blob/master/docker/README.md#create-torchserve-docker-image)
+
+Example:
+```
+./build_image.sh -bi nvidia/cuda:11.7.0-devel-ubuntu20.04  -g -cv cu117 -t pytorch/torchserve:latest-gpu
```

Comment:
Does Deepspeed have restriction for using with CUDA 11.7?

Reply:
No, it doesn't

Reply:
Removed the doc update for deepspeed. Will create a separate PR for this. I noticed torchserve crashing. Don't want to block this PR